metabase/metabase
# Implement funnel chart renderer

Epic: https://github.com/metabase/metabase/issues/18676

Add funnel static visualization renderer. There is no BE yet.

<img width="584" alt="Screen Shot 2021-11-26 at 02 10 14" src="https://user-images.githubusercontent.com/14301985/143503864-bd9edf59-051c-45f5-a9e6-f2fd01f55db5.png">

### How to verify

- Run MB and open the [internal page](http://localhost:3000/_internal/static-viz)
- Scroll down to the funnel static viz
- Ensure it renders correctly
